parameters/navigation/global_costmap_gmapping.yaml

```diff
@@ -5,6 +5,7 @@ robot_base_frame: /base_footprint
 rolling_window: false
 track_unknown_space: false
 resolution: 0.05
+robot_radius: 0.24
```

Comment:
I assume this was measured. Or was it a guestimate

Reply:
Measured and compared to values used by you and @JosjaG